lib/urldata.h

```diff
@@ -576,6 +576,9 @@ struct ConnectBits {
   bool proxy_ssl_connected[2]; /* TRUE when SSL initialization for HTTPS proxy
                                   is complete */
   bool socksproxy_connecting; /* connecting through a socks proxy */
+  bool needs_update; /* a non-blocking socket was created and a
```

Comment:
No need to mention non-blocking here, since this should work with any type of socket.